db/projects/neo/project.yaml

```diff
@@ -0,0 +1,170 @@
+name: NEO
+web:
+  - 'https://neo.org/'
+start_date: '2014-02-01'
+description: 'NEO is a blockchain, originally known as AntShares, sometimes called as chinese Ethereum. It is a first open source platform for smart contracts started in China.'
+ledgers:
+  - id: neo
+    name: NEO
+    type: blockchain
+    networks:
+      - id: main
+        type: main
+        name: Main Net
+        algorithm: dBFT
+        proof_type: PoS
+        tools:
+          - type: explorer
+            url: 'https://neotracker.io'
+          - type: explorer
+            url: 'https://neoscan.io/'
+          - type: explorer
+            url: 'http://monitor.cityofzion.io/'
+          - type: explorer
+            url: 'https://scan.nel.group/#mainnet'
+          - type: explorer
+            url: 'http://state.otcgo.cn/index.html?network=mainnet&idx=0'
+          - type: explorer
+            url: 'https://blolys.com/#/mainnet'
+      - id: test
+        type: test
+        name: TestNet
+        tools:
+          - type: explorer
+            url: 'http://monitor.cityofzion.io/'
+          - type: explorer
+            url: 'https://scan.nel.group/#testnet'
+          - type: explorer
+            url: 'https://neoscan-testnet.io/'
+          - type: explorer
+            url: 'http://state.otcgo.cn/?network=testne&idx=0'
+assets:
+  - id: neo
```

Comment:
Maybe we can include `decimals: 0` to NEO asset? I think it's very important information that NEO unit cannot be subdivided.
> The minimum unit of NEO is 1 and tokens cannot be subdivided.

Reply:
Why not, it's true that it's specific for NEO asset, so it should be mantioned. I have added:

decimals: 0